cms/djangoapps/contentstore/views/tests/test_item.py

```diff
@@ -1144,6 +1154,58 @@ def test_move_logging(self, mock_logger):
             insert_at
         )
 
+    @ddt.data(ModuleStoreEnum.Type.mongo, ModuleStoreEnum.Type.split)
+    def test_move_and_discard_changes(self, store_type):
+        """
+        Test that discard changes operation works as expected after a move operation bring back the component back.
```

Comment:
`Verifies that discard changes operation brings moved component back to source location and remove the component from target location`.